src/main.rs

```diff
@@ -91,7 +91,7 @@ fn go_spacemesh_default_path() -> &'static str {
 }
 
 fn backup_or_fail(file_path: &PathBuf) -> () {
-  if file_path.exists() {
+  if file_path.try_exists().unwrap_or(false) {
```

Comment:
I'm not sure it's a good idea to unwrap `Err` to `false`. An error could mean a lack of permissions (https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.try_exists). In such a situation, it's probably best to fail and to advice the user to verify rather than just not back the files up. Wdyt?

Reply:
Oops. You're right! 👍